ReactWindows/ReactNative.Net46/UIManager/UIManagerModule.cs

```diff
@@ -48,8 +49,14 @@ public partial class UIManagerModule : ReactContextNativeModuleBase, ILifecycleE
             _window = window;
             _eventDispatcher = new EventDispatcher(reactContext);
             _uiImplementation = uiImplementation;
+            _uiImplementation.EventDispatcher = _eventDispatcher;
```

Comment:
Injecting the EventDispatcher into the UIImplementation object instead of passing the EventDispatcher in every call to UIImplementation methods...